packages/module/src/ConsoleInternal/module/k8s/types.ts

```diff
@@ -42,4 +42,6 @@ export type ObjectMetadata = {
   ownerReferences?: OwnerReference[];
   resourceVersion?: string;
   uid?: string;
+  language?: string;
+  countryCode?: string;
```

Comment:
Can this just be `country`? And I would suggest adding comments (I know we don't have any yet) for these new fields. And perhaps also add a `locale` field.

Also, I'm thinking we should just let consumers allow to put any fields in metadata, since this will be project specific. 
https://github.com/patternfly/patternfly-quickstarts/blob/7d124e673f35ff58f1d921695e08b0ca90ce2dea/packages/module/src/utils/quick-start-types.ts#L9
We can change the type here to `metadata: ObjectMetadata | object;` I believe.